src/main/kotlin/ar/edu/unahur/obj2/tareas/Tarea.kt

```diff
@@ -1,3 +1,54 @@
 package ar.edu.unahur.obj2.tareas
 
-class Tarea
+class Tarea(val horasEstimadas:Int,val responsable : Responsable,
+            val costoInfraestructura: Double){
+    val empleados = mutableListOf<Empleado>()
+
+    fun asignarEmpleado(empleado: Empleado){
+        empleados.add(empleado)
+    }
+    fun horasNecesarias(): Int {
+        return horasEstimadas / empleados.size
+    }
+    fun costoTarea(): Double {
+        return this.horasNecesarias() * empleados.sumByDouble { it.sueldoPorHora } +
+                horasEstimadas * responsable.sueldo + this.costoInfraestructura
+    }
+    /*Costo de Tarea =
+  (Horas Estimadas / Nro. de Empleados) * Sueldo por hora de Cada Empleado +
+  Horas Estimadas * Sueldo del Responsable +
+  costo de infraestructura*/
+
+    fun nominaDeEmpleados(): MutableList<Trabajador> {
+        val nomina = mutableListOf<Trabajador>()
+        nomina.addAll(this.empleados)
+        nomina.add(this.responsable)
+        return nomina
+    }
+}
+open class Trabajador {}
+class Empleado(val sueldoPorHora: Double): Trabajador(){}
+class Responsable(val sueldo: Double) : Trabajador(){}
+
+
+class TareaDeIntegracion(val tareas: List<Tarea>,val responsable: Responsable){
+    fun horasNecesarias(): Int {
+        return tareas.sumBy { it.horasNecesarias() } / 8 + tareas.sumBy { it.horasNecesarias() }
+    }
+
+    fun costoTareas(): Double{
+        return tareas.sumByDouble { it.costoTarea() }
+    }
+    fun plusResponsable(): Double {
+        return this.costoTareas() * 0.03
+    }
+    fun costoTotal():Double{
+        return this.costoTareas()+ this.plusResponsable()
+    }
```

Comment:
Todo esto se podría haber hecho de una.. 
```suggestion
    fun costoTarea() = //Para no romper el polimorfismo
    tareas.sumBy {it.costo() } * 1.03
```